### PR TITLE
chore(control plane): add resource constraints

### PR DESCRIPTION
**Pull request by paulyoong**
Add resource constraints for the control plane components. This has
been set to reasonable estimates based on a system with 109 volumes
that is handling many concurrent REST requests.

Resolves: CAS-1200

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]
Could you please bump up the resources in deploy/terraform/simple.tfvars?

File: deploy/rest-deployment.yaml [hunk 1]
```diff
@@ -36,6 +36,13 @@ spec:
...
       containers:
         - name: rest
+          resources:
+            limits:
+              cpu: 100m
+              memory: 64Mi
+            requests:
+              cpu: 50m
+              memory: 32Mi
           image: mayadata/mcp-rest:develop
```
[the remark this answers is not in the document]
Comment (paulyoong) [reply deploy/rest-deployment.yaml]: Good idea. Done.

**Comment by paulyoong** [comment]
> Could you please bump up the resources in deploy/terraform/simple.tfvars?

Done.

**Comment by paulyoong** [comment]
bors merge

**Pull request merged**